package.json

```diff
@@ -103,5 +102,8 @@
     "webpack": "^4.44.1",
     "webpack-bundle-analyzer": "^3.8.0",
     "webpack-cli": "^3.3.12"
+  },
+  "optionalDependencies": {
+    "@chainsafe/filsnap-adapter": "^2.1.1"
```

Comment:
Ok I see from a conversation with Marin that he recommends it as optional since this is apparently used to allow people to use it in the browser.